src/test/java/de/doubleslash/usb_led_matrix/view/ConfigurationViewTest.java

```diff
@@ -124,39 +125,58 @@ void shouldLeaveTimeoutWhenColorChanges() {
    @Test
    void shouldTurnOffWhenCurrentTimeEqualsTimeout() {
       // Given
-      final LocalTime currentTime = LocalTime.of(21, 0);
+      final LocalTime currentTime = LocalTime.of(20, 0);
+
       configurationView.setUsbAdapter(usbAdapter);
       CommandLineOptions.setTimeoutInMinutes(60);
-      configurationView.timeLightsTurnedOffNow = LocalTime.of(20, 0);
+      configurationView.timeLightsTurnedOffNow = LocalTime.of(21, 0);
       // When
-      configurationView.turnOffAutomaticallyIfNeeded(currentTime);
+      configurationView.turnOffAutomaticallyIfNeeded(currentTime, LocalDate.now());
       // Then
-      verify(usbAdapter).updatePixel(Color.BLACK);
+      verify(usbAdapter, never()).updatePixel(Color.BLACK);
    }
 
    @Test
    void shouldNotTurnOffWhenCurrentTimeDoesNotReachTimeout() {
       // Given
       final LocalTime currentTime = LocalTime.of(19, 0);
+
       configurationView.setUsbAdapter(usbAdapter);
       CommandLineOptions.setTimeoutInMinutes(60);
       configurationView.timeLightsTurnedOffNow = LocalTime.of(20, 0);
       // When
-      configurationView.turnOffAutomaticallyIfNeeded(currentTime);
+      configurationView.turnOffAutomaticallyIfNeeded(currentTime, LocalDate.now());
       // Then
       verify(usbAdapter, never()).updatePixel(Color.BLACK);
    }
 
+   @Test
+   void shouldGoOutIfTheTimeIsExceededOnTheNextday() {
+      // Given
+      final LocalTime currentTime = LocalTime.of(23,42,39,91);
+      configurationView.setUsbAdapter(usbAdapter);
+      CommandLineOptions.setTimeoutInMinutes(60);
+      LocalDate startDate = LocalDate.now().plusDays(1);
+      configurationView.timeLightsTurnedOffNow = LocalTime.of(01,42,33,747380700);
```

Comment:
the test setup does only setup a time when it should turn off - but not a date. seems it is not known at what date it should turn off? hard to understand (or i am just confused)

Reply:
Done.